packages/bezier-react/src/components/Banner/Banner.tsx

```diff
@@ -96,14 +96,18 @@ export const Banner = forwardRef<HTMLDivElement, BannerProps>(function Banner(
         <div className={styles.Center}>
           {isBezierIcon(icon) ? (
             <Icon
-              className={styles.Icon}
+              className={classNames({
+                [styles.Icon]: !iconColor,
+              })}
```

Comment:
```suggestion
              className={classNames(
                !iconColor && styles.Icon,
              )}
```

문자열이 객체 생성 오버헤드가 없어서 성능 이점이 더 있는 거로 알고있어요